<launch>
    <!-- stream URL components -->
    <arg name="rtsp_stream_url" default="rtsp://192.168.0.5:5005/routecam" />
    <arg name="image_topic" default="/ip_camera/image_raw" />
    <arg name="image_scale" default="1.0" />
    <!-- Keep this between 0.0 and 1.0 -->

    <!-- Launch node -->
    <node pkg="ip_camera_publisher" name="ip_camera_publisher" type="route_cam.py">
        <param name="rtsp_stream_url" value="$(arg rtsp_stream_url)" />
        <param name="image_topic" value="$(arg image_topic)" />
        <param name="image_scale" value="$(arg image_scale)" />
    </node>

</launch>